frontend/public/help/en/help.issue.detail.md

```diff
@@ -0,0 +1,9 @@
+---
+title: What is 'Issue' ?
+---
+
+- Issue represents a specific collaboration activity between developer and DBA such as creating a database, altering a schema. It's similar to the issue concept in other issue management tools.
+
+#### Learn more
```

Comment:
Should be using by order `##`, `###`, ...

Reply:
the visual result will not be so good... as we have a big title up there ...